Source/JavaScriptCore/runtime/JSCBytecodeCacheVersion.cpp

```diff
@@ -66,7 +66,7 @@ uint32_t computeJSCBytecodeCacheVersion()
         }
         cacheVersion.construct(0);
         dataLogLnIf(JSCBytecodeCacheVersionInternal::verbose, "Failed to get UUID for JavaScriptCore framework");
-#elif OS(UNIX) && !PLATFORM(PLAYSTATION)
+#elif OS(UNIX) && !PLATFORM(PLAYSTATION) && !OS(HAIKU)
```

Comment:
Don't Haiku have dlopen etc? Can you implement them?

Reply:
If I remember correctly, the current release of Haiku is missing dl_iterate_phdr, it will be there in the next release (beta 6) and then this can be removed.